packages/legend-extension-dsl-data-space/src/graphManager/action/analytics/DSL_DataSpace_DataSpaceAnalysis.ts

```diff
@@ -23,6 +23,10 @@ import type {
 import { uuid } from '@finos/legend-shared';
 import type { DataSpaceSupportInfo } from '../../../graph/metamodel/pure/model/packageableElements/dataSpace/DSL_DataSpace_DataSpace.js';
 
+export const DATASPACE_ANALYTICS_ARTIFACT_EXTENSION_KEY = 'dataSpace-analytics';
```

Comment:
I think we can move these into `V1_DSL_DataSpace_PureGraphManagerExtension`. See my comment there